### PR TITLE
Refactor stanford_sites_helper to be quicker

### PR DESCRIPTION
**Pull request by jbickar**
# READY FOR REVIEW

# Summary
- Cuts down the running time of the `stanford_sites_helper.feature` by about 80%

# Needed By (Date)
- Whenevah

# Criticality
- How critical is this PR on a 1-10 scale? 2
- But it does help get our Travis runs under the 49 minute threshold

# Steps to Test

1. Check out this branch
2. Run `sites/uat/features/high-value/stanford_sites_helper.feature`
3. Ensure that it passes
4. Review the diff to see that nothing was missed

# Affected Projects or Products
- https://github.com/SU-SWS/Stanford-Drupal-Profile

# Associated Issues and/or People
- Anyone who should be notified? ( @kbrownell )

### CODE REVIEW

File: sites/uat/features/high-value/stanford_sites_helper.feature [hunk 1]
```diff
@@ -5,33 +5,27 @@ Feature: Stanford Sites Helper Module
   I want to ensure the Stanford Sites Helper module is working properly
 
   @api @safe @deploy @javascript
-  Scenario Outline: Homepage text content
```
Comment (sherakama) [inline sites/uat/features/high-value/stanford_sites_helper.feature:8]: Hmmm... interesting. Scenario Outlines are slow. Is this noted anywhere out there in the interwebs?

Comment (jbickar) [reply sites/uat/features/high-value/stanford_sites_helper.feature:8]: For each line, it goes through and does each step in the outline:
1. Given I am an anonymous user
2. And I am on the homepage
3. Then I should see " Once you have logged in, you will be able to remove this text from your homepage" in the "Content Body" region
1. Given I am an anonymous user
2. And I am on the homepage
3. Then I should see "Create a new page on your site" in the "Content Body" region

etc.

Not so bad for this one, but for the further ones, where it's creating a new user, logging in as that user, waiting for AJAX to finish, it's doing that for every line in the "Examples" table.

Comment (jbickar) [reply sites/uat/features/high-value/stanford_sites_helper.feature:8]:
```
Before refactoring (local):

24 scenarios (24 passed)
116 steps (116 passed)
5m12.76s (20.91Mb)

After refactoring (local):

5 scenarios (5 passed)
88 steps (88 passed)
1m25.16s (20.66Mb)
```

Comment (sherakama) [reply sites/uat/features/high-value/stanford_sites_helper.feature:8]: That makes sense. Add that to the lessons learned of the week for staff.

**Pull request merged**